lib/services/graphql_config.dart

```diff
@@ -32,10 +32,19 @@ class GraphqlConfig {
     authClient();
   }
 
+  static final WebSocketLink _webSocketLink = WebSocketLink(
+    'ws://sriram.dscnitrourkela.org/graphql',
```

Comment:
Hey, was this link a test link? Should it be replaced before merging?

Reply:
Yeah, this was a test link. I'll make the needed changes.